packages/pwa-kit-extension-sdk/src/types.ts

```diff
@@ -46,13 +46,6 @@ export interface GetRoutesParams {
     locals: Record<string, any>
 }
 
-/*
- * Same as react-router's RouteProps, but now supports `componentName` if the route is meant to be serializable
- */
-export type RouteProps = _RouteProps & {
```

Comment:
FYI you'll need to replace all references to this RouteProps.. with the one from react-router instead.